docs/topics/request-response.rst

```diff
@@ -364,6 +365,20 @@ started, i.e. HTTP message sent over the network. This meta key only becomes
 available when the response has been downloaded. While most other meta keys are
 used to control Scrapy behavior, this one is supposed to be read-only.
 
+.. reqmeta:: download_slot
+
+download_slot
+----------------
+
+By default, it stores the hostname of the website to which request is destined.
+Scrapy uses slots as queues to process requests and `download_slot` represent the slot
+in which that request is going to process. If a slot doesn't exist then Scrapy creates
+it and uses it to process requests associated with it. A slot adds different constraints
+to the requests to tune up the concurrency and add delays e.g. :setting:`DOWNLOAD_DELAY`,
+:setting:`CONCURRENT_REQUESTS_PER_DOMAIN`, :setting:`CONCURRENT_REQUESTS_PER_IP` etc.
+By default all the requests related to a domain are processed in single slot.
+But you can set this key to whatever you like when sending the request.
```

Comment:
A few suggestions:
- Start by explaining what this request meta key does, not its default value.
- Use paragraphs.
- “If a slot doesn't exist then Scrapy creates it and uses it to process requests associated with it.” feels like an implementation detail, introduced before, from the reading so far, I could understood what slots are for.
- It is not clear to me, from reading, how those settings relate to slots. What happens if I give a value to each of those 3 settings, and then I use a custom download slot?
- It may be helpful to cover some of the use cases of this feature.